src/main/kotlin/org/radarbase/push/integration/common/filter/CorsFilter.kt

```diff
@@ -0,0 +1,22 @@
+package org.radarbase.push.integration.common.filter
+
+import java.io.IOException
+import javax.ws.rs.container.ContainerRequestContext
+import javax.ws.rs.container.ContainerResponseContext
+import javax.ws.rs.container.ContainerResponseFilter
+import javax.ws.rs.ext.Provider
+
+@Provider
+class CorsFilter : ContainerResponseFilter {
```

Comment:
CorsFilter is included in radar-jersey. Include in the classes Val of the resource enhancer `ConfigLoader.Filters.CorsFilter`

Reply:
I checked that before adding this. The radar-jersey one has extra allowed methods that we don't have (like PUT, GET, DELETE, etc). Is it still ok to include it?